module/actor/witcherActor.js

```diff
@@ -25,7 +24,266 @@ export default class WitcherActor extends Actor {
         }
       });
     }
+  }
+
+  updateDerived() {
+    const stats = this.system.stats;
+    const base = Math.floor((stats.body.current + stats.will.current) / 2);
+    const baseMax = Math.floor((stats.body.max + stats.will.max) / 2);
+    const meleeBonus = Math.ceil((stats.body.current - 6) / 2) * 2;
+
+    let intTotalModifiers = this.getAllModifier("int").totalModifiers;
```

Comment:
I think its a good practive to only use const. The code is pretty hard to read, because it continuosly changes the variable values and definitions. It also has a lot of duplicated code. (See following comments...)

Reply:
https://github.com/witchertrpg-foundryvtt/TheWitcherTRPG/issues/61